### ISSUE TITLE
Windows: [error] this action cannot be run on win32

### ISSUE THREAD

**Issue by Huliiiiii** [edgedb/edgedb-editor-plugin#43]
Output of EdgeDB Language Client:
```
2024-09-13 21:21:03.704 [info] Extension activated.
2024-09-13 21:21:03.704 [error] this action cannot be run on win32
```

**Comment by aljazerzen** [comment]
I assume you are using Windows? To be honest the language server was not tested on Windows yet.

I looks like I'll have to get a windows machine :sweat:

**Comment by gubbsjuk** [comment]
@aljazerzen Any update here?

**Comment by nerdoza** [comment]
This seems to be a wider spread issue across multiple implementations of the `gel-cli` wrappers. Using `gel` purely from the command line in Windows works fine, but the same calls made from wrapped contexts like `npx` or this plugin result in the same **"this action cannot be run on win32"** error.

**Comment by mmastrac** [comment]
@nerdoza There are likely two issues here -- the gel language server doesn't currently support windows.

As for the `npx` issue, I believe this is just missing Windows support in gel-js here (@scotttrinh, can we add support for platform == windows?):

https://github.com/geldata/gel-js/blob/7681512862a5bec545c47b6cd1ca7a9608e78d81/packages/gel/src/cli.mts#L304

**Comment by mmastrac** [comment]
@nerdoza was the language server functional on Windows? I wasn't sure if we ever supported it and it looks like perhaps we started showing errors where we might have been silently ignoring them before.

We don't want to break your Windows development workflow for sure. I'm not sure _how_ this got broken as we believed it was all working as expected, but please feel free to file issues with whatever information you can provide. We will certainly try to fix any regressions.

**Comment by aljazerzen** [comment]
No, we currently don't support language server on windows. I'm not sure if we even build binaries for it.
